tests/monitors/kubernetes_volumes/kubernetes_volumes_test.py

```diff
@@ -15,7 +15,10 @@ def test_kubernetes_volumes_in_k8s(k8s_cluster):
          kubeletAPI:
            skipVerify: true
            authType: serviceAccount
+         extraMetrics:
```

Comment:
Why do you need to put anything in `extraMetrics` since this monitor is configured to send all metrics?

Reply:
I changed sendAll to false and made the original metrics default, the new ones will be non-default